src/lib/server/embeddingEndpoints/embeddingEndpoints.ts

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
it makes sense to put this file inside https://github.com/huggingface/chat-ui/tree/main/src/lib/types folder